src/views/layouts/master.twig

```diff
@@ -112,6 +112,7 @@
   
     <script defer src="{{ 'product-card.js'|asset }}"></script>
     <script defer src="{{ 'main-menu.js'|asset }}"></script>
+    <script defer src="{{ 'wishlist-card.js'|asset }}"></script>
```

Comment:
add new block, call it `head_start` for example and pass this from wishlist page